### ISSUE TITLE
Select: failed validations do not make the helper text "red" & force-display it

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2263]
When a validation fails, the helper text should become red. 
The helper text is such cases typically explain what is wrong, e.g. it can state: `This is a required field.`

https://user-images.githubusercontent.com/35954987/234211304-30287a20-2a4b-4624-a37f-89c60480b5ab.mov

Also, the current helper line in limel-select always has `aria-hidden="true"`, which is wrong.
